### PR TITLE
Scaffold project

### PR DESCRIPTION
**Pull request by aszecowka**
- use repository template from kyma-project/community repository
- initialize kubebuilder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
-# octopus
-The test runner for acceptance tests
+# Octopus
+
+## Overview
+
+Octopus is a testing framework that allows running tests defined as Docker images on a running cluster.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Octopus is a testing framework that allows you to run tests defined as Docker images on a running cluster.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+Think about Octopus as a `helm test` on steroids.
```
Comment (klaudiagrz) [inline README.md]: Seriously? 🙂 🐙 

As much as I love the metaphor, I think we should skip it in the official README.

**Review by klaudiagrz — COMMENTED** [review]
Also: your new repository lacks two documents:
- CODEOWNERS and
- LICENSE
Find their templates [here](https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/README.md).

File: README.md (hunk 2 above)
Comment (aszecowka) [reply README.md]: It was the best sentence :(

**Review by mszostok — CHANGES_REQUESTED** [review]
Two things:
- please use kubebuilder in version 1.0.7 (we have buildpack created for it)
- please adjust the `/config` folder, see this issue for more info: https://github.com/kubernetes-sigs/kubebuilder/issues/595

you can use their solution or this from prow-addons-ctrl-manager

File: docs/LICENSE [hunk 3]
```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```
Comment (klaudiagrz) [inline docs/LICENSE]: Why is this license placed in the `docs` folder? There is also the other license in the root folder. Which one is the right one? Please clarify that.

Comment (aszecowka) [reply docs/LICENSE]: removed

**Review by mszostok — APPROVED** [review]

**Pull request merged**